src/storage-client/src/client.rs

```diff
@@ -354,6 +354,65 @@ impl PackableStats for SinkStatisticsUpdate {
     }
 }
 
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub enum Status {
```

Comment:
I think this should have some documentation, explaining that its a "kind" enum for statuses tracked by the health operator

Reply:
Done